test/Connector.t.sol

```diff
@@ -338,39 +340,36 @@ contract ConnectorTest is Test {
         connector.updateMember(poolId, trancheId, destinationAddress, validUntil);
 
         bytes9 encodedDomain = ConnectorMessages.formatDomain(ConnectorMessages.Domain.Centrifuge);
-        connector.transfer(poolId, trancheId, encodedDomain, destinationAddress, amount);
+        connector.incomingTransfer(poolId, trancheId, 1, destinationAddress, amount);
 
         (address token,,,,) = bridgedConnector.tranches(poolId, trancheId);
         assertEq(ERC20Like(token).balanceOf(destinationAddress), amount);
     }
 
-    function testTransferTrancheTokensToEVM(
+    function testTransferTrancheTokensFromCentrifugeWithoutMemberFails(
```

Comment:
Last time I saw a name this long Java 8 was the next big thing 😆

Reply:
Me too actually. Although that I prefer /// comments over long names for readability but this is fine 👍